tests/validators/test_union.py

```diff
@@ -409,8 +409,8 @@ def test_strict_reference():
             ref='tuple-ref',
         )
     )
-    assert 'strict_required:true' in plain_repr(v)
-    assert 'ultra_strict_required:true' in plain_repr(v)  # since "float" schema has ultra-strict behaviour
+    # assert 'strict_required:true' in plain_repr(v)
+    # assert 'ultra_strict_required:true' in plain_repr(v)  # since "float" schema has ultra-strict behaviour
```

Comment:
are you sure this needs commenting out?

Reply:
It didn't! Thanks for pointing it out, fixed